src/arch/xtensa/drivers/timer.c

```diff
@@ -144,3 +146,5 @@ int64_t arch_timer_set(struct timer *timer, uint64_t ticks)
 
 	return ret;
 }
+
+#endif
```

Comment:
Is this excluding the whole file ? If so, we could just remove it from build in Zephyr mode.